app/models/taxonomy/expanded_taxonomy.rb

```diff
@@ -80,11 +85,24 @@ def expand_parent_nodes(start_node:, parent:)
     end
 
     def tree_node_based_on(content_item)
+      if content_item['content_id'] == GovukTaxonomy::ROOT_CONTENT_ID
+        home_page_linked_content_item
+      else
+        GovukTaxonomyHelpers::LinkedContentItem.new(
+          internal_name: content_item.dig('details', 'internal_name'),
+          title: content_item.fetch('title'),
+          base_path: content_item.fetch('base_path'),
+          content_id: content_item.fetch('content_id')
+        )
+      end
+    end
+
+    def home_page_linked_content_item
       GovukTaxonomyHelpers::LinkedContentItem.new(
-        internal_name: content_item.dig('details', 'internal_name'),
-        title: content_item.fetch('title'),
-        base_path: content_item.fetch('base_path'),
-        content_id: content_item.fetch('content_id')
+        internal_name: 'Root of the taxonomy',
+        title: 'Root Taxon',
```

Comment:
Personally I'd call this the "Homepage", but I guess that's something that Sona might have an opinion on as well?